pkg/volume/portworx/portworx_util.go

```diff
@@ -182,6 +186,27 @@ func (util *PortworxVolumeUtil) UnmountVolume(u *portworxVolumeUnmounter, mountP
 	return nil
 }
 
+func (util *PortworxVolumeUtil) ResizeVolume(spec *volume.Spec, newSize resource.Quantity, volumeHost volume.VolumeHost) error {
+	driver, err := util.getPortworxDriver(volumeHost, false /*localOnly*/)
+	if err != nil || driver == nil {
+		glog.Errorf("Failed to get portworx driver. Err: %v", err)
+		return err
+	}
+
+	vols, err := driver.Inspect([]string{spec.Name()})
+	if err != nil {
+		return err
+	}
+
+	if len(vols) != 1 {
+		return fmt.Errorf("failed to inspect Portworx volume: %s. Found: %d volumes", spec.Name(), len(vols))
+	}
+
+	vol := vols[0]
```

Comment:
only the first volume will be resized?

Reply:
Yes, because we inspected one volume above. So only one volume is expected is the response. Line 200 also explicitly makes that check. `ResizeVolume` is a per-volume call.